xarray/util/generate_aggregations.py

```diff
@@ -259,6 +257,7 @@ def __init__(
         docref_description,
         example_call_preamble,
         definition_preamble,
+        template_see_also,
```

Comment:
Lets remove this since its always the same.

Reply:
done